src/platform/user/profile/vet360/components/MobilePhone.jsx

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
-import environment from 'platform/utilities/environment';
+import { connect } from 'react-redux';
 import PhoneField from './PhoneField';
 import ReceiveTextMessages from '../containers/ReceiveTextMessages';
 import { FIELD_NAMES } from '../constants';
+import { profileShowNotifications } from '../selectors';
 
-export default function MobilePhone() {
-  return (
-    <>
-      <PhoneField
-        title="Mobile phone number"
-        fieldName={FIELD_NAMES.MOBILE_PHONE}
-      />
-      {!environment.isProduction() && (
-        <ReceiveTextMessages
+class MobilePhone extends React.Component {
```

Comment:
this doesn't need to be converted to a React.Component. I think you could just remove the `export default` and pass the props arg to the `MobilePhone` function...

Reply:
I want to leave it this way because I don't want to deal with passing this flag in as a prop JUST to mobile phone.